### PR TITLE
fix assertion error, not show remove liquidity when user don't have l…

### PR DESCRIPTION
**Pull request by SzymonZiebaNeti**
…iquidity

### CODE REVIEW

File: src/components/Wallet/Exchange/ExchangeShowMore/index.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ function ExchangeShowMore({
                   {' '}
                   {new BN(lpTokensBalance).mul(BN_HUNDRED).div(new BN(liquidity)).toString()}
                   % (
-                  {lpTokensBalance}
+                  {lpTokensBalance.toString()}
```
Comment (kacperzuk-neti) [inline src/components/Wallet/Exchange/ExchangeShowMore/index.js:52]: Hmmm... lpTokensBalance is a BN here, right? I'd worry that `.toString()` may return hex for some values here. Maybe use formatAssets with decimals=0 instead?
Also, what values are seen here in practice? I'd worry that these are gigantic number, while at the same time it doesn't really make sense to put arbitrary decimals on it... Is it needed at all? Just keeping percent seems enough.

File: src/components/Wallet/Exchange/ExchangeShowMore/index.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ function ExchangeShowMore({
                   {' '}
                   {new BN(lpTokensBalance).mul(BN_HUNDRED).div(new BN(liquidity)).toString()}
```
Comment (kacperzuk-neti) [inline src/components/Wallet/Exchange/ExchangeShowMore/index.js:50]: `lpTokensBalance` already is a BN now?

File: src/components/Wallet/Exchange/ExchangeShowMore/index.js [hunk 3]
```diff
@@ -80,7 +80,7 @@ function ExchangeShowMore({
 
         </div>
         <div className={styles.liquidityButtons}>
-          {!isReservedDataEmpty
+          {!isReservedDataEmpty && (lpTokensBalance !== 0 || !liquidity)
```
Comment (kacperzuk-neti) [inline src/components/Wallet/Exchange/ExchangeShowMore/index.js:83]: `lpTokensBalance !== 0` probably won't work - `lpTokensBalance` is now an object (BN probably).

**Review by kacperzuk-neti — CHANGES_REQUESTED** [review]

File: src/components/Wallet/Exchange/ExchangeShowMore/index.js (hunk 1 above)
Comment (DorianSternVukotic) [reply src/components/Wallet/Exchange/ExchangeShowMore/index.js:52]: yeah just keep percent then

**Review by kacperzuk-neti — APPROVED** [review]

**Pull request merged**